REFERENCE.md

```diff
@@ -1200,6 +1209,14 @@ Data type: `Boolean`
 
 Default value: ``false``
 
+##### `scrape_host`
+
+Data type: `Optional[Stdlib::Host]`
+
+
```

Comment:
this is fairly sparse on documentation

Reply:
but isn't that out of scope for this PR?

Reply:
fair